containers/podman-desktop
# fix: Do not use template icon for Windows and Linux

### What does this PR do?
Template icons are only working for macOS (others are not supporting it)
https://www.electronjs.org/docs/latest/api/native-image#template-image

Use icons based on the selected theme for Windows (usually the bar is following the theme color)

While on Linux, most of the time, on Gnome for example the bar is black) so always use white icons fixes https://github.com/containers/podman-desktop/issues/467

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/436777/191958311-7d79f2a2-4802-4d6b-a2d0-3b997cd826e5.png)


### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop/issues/467

### How to test this PR?

Launch on Linux and check the tray icon

note: Gnome is not displaying tray icons by default so this extension needs to be installed https://extensions.gnome.org/extension/615/appindicator-support/

Change-Id: Ic7e593c6b2ffdcff53d465664d7c960ba78e03b8
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
